conans/client/proxy.py

```diff
@@ -112,6 +112,15 @@ def handle_package_manifest(self, package_ref, installed):
             remote = self._registry.get_ref(package_ref.conan)
             self._manifest_manager.check_package(package_ref, remote)
 
+    def get_recipe_sources(self, conan_reference):
+        export_path = self._client_cache.export(conan_reference)
+        try:
+            current_remote, _ = self._get_remote(conan_reference)
+        except ConanException:
+            pass
```

Comment:
This is a bit weird, for "conan export" packages it will always be a ConanException, because the remote is not set. But it can also hide inconsistencies, like the remote is not set but it should be set to be able to retrieve the sources.

Solution could be to bring here the check for existence of the ".conan_export_sources" folder, so it wouldn't be fully encapsulated into the ``RemoteManager``